helm/handbook/templates/service.yaml

```diff
@@ -11,3 +11,17 @@ spec:
       targetPort: http
   selector:
     app: handbook
+---
+kind: Service
+apiVersion: v1
+metadata:
+  namespace: intranet
+  name: staticjscms-hugo-standalone
+  labels:
+    app: staticjscms-hugo-standalone
+spec:
+  ports:
+    - port: 8080
+      targetPort: http
+  selector:
+    app: staticjscms-hugo-standalone
```

Comment:
Is this tested? As far as I can find no deployment with this label exists. You added the second container to the same pod  (in one deployment) but you are referencing a label here which does not have a matching deployment. So IMO this selector would currently select nothing.